packages/nodejs/src/scope.ts

```diff
@@ -221,30 +227,85 @@ export class ScopeManager {
     // prevent re-wrapping
     contextWrapper[WRAPPED] = true
 
-    // explicitly inherit the original function's length, because it is
-    // otherwise zero-ed out
+    // explicitly inherit the original function's length and name
     Object.defineProperty(contextWrapper, "length", {
-      enumerable: false,
       configurable: true,
-      writable: false,
       value: fn.length
     })
 
+    Object.defineProperty(contextWrapper, "name", {
```

Comment:
What problem does copying over the name property solve?
Is this something we can test for?

Reply:
There definitely should be a test!

It doesn't really directly solve a problem, but if someone's debugging their application with good ol' `console.log`, the wrapper function will have the same name as the wrapped function, instead of being an unnamed function. I'm happy to remove this commit, it's a minor improvement and it's not really related to the actual fix. I just spent a lot of time staring at various `console.log` statements while trying to figure out the fix.